src/operations/create_collection.ts

```diff
@@ -98,6 +100,10 @@ export interface CreateCollectionOptions extends CommandOperationOptions {
   changeStreamPreAndPostImages?: { enabled: boolean };
 }
 
+/* @internal */
+const INVALID_QE_VERSION =
+  'Driver support of Queryable Encryption is incompatible with server. Upgrade server to use Queryable Encryption.';
```

Comment:
Do we want to mention 7.0 as a version here? That might be good because users may think that QE was introduced in 6.0 (and they wouldn’t be _wrong_ about that, either).

Reply:
I agree with mentioning 7.0 here

Reply:
Updated to mention 7.0.